test/unit-test/mqtt_agent_utest.c

```diff
@@ -143,7 +150,7 @@ static bool stubReceive( AgentMessageContext_t * pMsgCtx,
 {
     Command_t ** pCommandToReceive = ( Command_t ** ) pBuffer;
 
-    *pCommandToReceive = pMsgCtx->pSentCommand;
+    *pCommandToReceive = globalMessageContext.pSentCommand;
```

Comment:
It doesn't really make sense to use the global context when we already have a context passed to us
```suggestion
    *pCommandToReceive = pMsgCtx->pSentCommand;
```